### ISSUE TITLE
va-file-input with password enhancement - Accessibility review

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#4999]
## Issue Description
We need to conduct a full accessibility review of the va-file-input with password enhancement component. The component can be viewed in Storybook here: https://4915-file-input-password-button--65a6e2ed2314f7b8f98609d8.chromatic.com/

## Tasks
- [x] Full accessibility review of the component is completed
- [ ] If mobile component validate on mobile devices the component accessibility behaviors match expectations
     - [ ] Screen readers
     - [ ] Text size changes
- [x] Create a ticket with descriptions of any issues found in a new ticket
    - [ ] If any problems are found, add screen recordings and/or screenshots to comments on this ticket that show the problem.
- [x] If necessary, discuss any accessibility issues found with the engineer who developed the component

## Acceptance Criteria
- [x] Component has been reviewed for accessibility issues
- [x] Any accessibility issues have been put in a new ticket

**Comment by jeana-adhoc** [comment]
Findings WIP

## Issue 1

I failed to document that on the initial password field display we need to connect the alert text to the input.  There's no indication to a screen reader user about the reason for the password. This is conveyed visually for users, but not for screen reader users.

Options:
1. Remove the warning all together and put the text in Hint text. 
2. Use aria-describedby on the password input to point to the alert.

**Comment by jeana-adhoc** [comment]
[Full testing Password on va-file-input documented here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/platform/design-system/components/va-file-input/tests/accessibility-tests.md#-encrypted-file--password-tests))

- **Known issue:** When browsing or swiping hidden screen reader text may be accessible and announced again (the file name)
- **Known issue:** When encountering an error in the password with Safari + VO: Focus goes back to the password field, and it announces "X characters Insertion at end of text. Password for this file (*Required) required invalid data secure edit text with autofill menu" it never announces an error or the error message. It never announces "Incorrect password. Try again or delete file." This is a known issue with the text input field. We are relying on the announcement of invalid data being a clue to users that there is an error in the password.
